### PR TITLE
Update authentication examples

### LINKED ISSUE #19252: Update basic auth implementation in examples.
The example [api-routes-apollo-server-and-client-auth](https://github.com/vercel/next.js/tree/canary/examples/api-routes-apollo-server-and-client-auth) has a better auth implementation (added in https://github.com/vercel/next.js/pull/13849) that the one in other auth examples with basic auth. Those are:

- with-passport
- with-magic
- with-passport-and-next-connect

Having a better basic auth implementation in examples is going to be very useful before landing https://github.com/vercel/next.js/pull/16277

### PR DESCRIPTION
**Pull request by lfades**
Improves the setup of our current auth examples that have basic auth to a more complete one, matching the updated `api-routes-apollo-server-and-client-auth`. Also updated the readme for `with-magic`.

Closes https://github.com/vercel/next.js/issues/19252

### CODE REVIEW

**Comment by ijjk** [comment]
# Stats from current PR

<details>
<summary><strong>Default Server Mode</strong> (Increase detected ⚠️)</summary>

<br/>

<details>
<summary><strong>General</strong></summary>

|  | vercel/next.js canary  | lfades/next.js update-basic-auth-implementation-in-examples-ch15325 | Change |
| - | - | - | - |
| buildDuration | 10.6s | 10.4s | -179ms |
| nodeModulesSize | 84.9 MB | 84.9 MB | ✓ |

</details>

<details>
<summary><strong>Page Load Tests</strong> Overall increase ✓</summary>

|  | vercel/next.js canary  | lfades/next.js update-basic-auth-implementation-in-examples-ch15325 | Change |
| - | - | - | - |
| / failed reqs | 0 | 0 | ✓ |
| / total time (seconds) | 2.385 | 2.391 | ⚠️ +0.01 |
| / avg req/sec | 1048.34 | 1045.54 | ⚠️ -2.8 |
| /error-in-render failed reqs | 0 | 0 | ✓ |
| /error-in-render total time (seconds) | 1.555 | 1.502 | -0.05 |
| /error-in-render avg req/sec | 1607.67 | 1664.28 | +56.61 |

</details>

<details>
<summary><strong>Client Bundles (main, webpack, commons)</strong></summary>

|  | vercel/next.js canary  | lfades/next.js update-basic-auth-implementation-in-examples-ch15325 | Change |
| - | - | - | - |
| 677f882d2ed8..4972.js gzip | 12.7 kB | 12.7 kB | ✓ |
| framework.HASH.js gzip | 39 kB | 39 kB | ✓ |
| main-f1a49fb..e45e.js gzip | 6.52 kB | 6.52 kB | ✓ |
| webpack-e067..f178.js gzip | 751 B | 751 B | ✓ |
| Overall change | 58.9 kB | 58.9 kB | ✓ |

</details>

<details>
<summary><strong>Legacy Client Bundles (polyfills)</strong></summary>

|  | vercel/next.js canary  | lfades/next.js update-basic-auth-implementation-in-examples-ch15325 | Change |
| - | - | - | - |
| polyfills-4b..e242.js gzip | 31 kB | 31 kB | ✓ |
| Overall change | 31 kB | 31 kB | ✓ |

</details>

<details>
<summary><strong>Client Pages</strong></summary>

|  | vercel/next.js canary  | lfades/next.js update-basic-auth-implementation-in-examples-ch15325 | Change |
| - | - | - | - |
| _app-3b0cf13..85f8.js gzip | 1.28 kB | 1.28 kB | ✓ |
| _error-6f635..c393.js gzip | 3.44 kB | 3.44 kB | ✓ |
| hooks-d4ffc3..9e0f.js gzip | 887 B | 887 B | ✓ |
| index-17468f..5d83.js gzip | 227 B | 227 B | ✓ |
| link-b618194..5477.js gzip | 1.61 kB | 1.61 kB | ✓ |
| routerDirect..924c.js gzip | 284 B | 284 B | ✓ |
| withRouter-7..c13d.js gzip | 284 B | 284 B | ✓ |
| Overall change | 8.01 kB | 8.01 kB | ✓ |

</details>

<details>
<summary><strong>Client Build Manifests</strong></summary>

|  | vercel/next.js canary  | lfades/next.js update-basic-auth-implementation-in-examples-ch15325 | Change |
| - | - | - | - |
| _buildManifest.js gzip | 321 B | 321 B | ✓ |
| Overall change | 321 B | 321 B | ✓ |

</details>

<details>
<summary><strong>Rendered Page Sizes</strong></summary>

|  | vercel/next.js canary  | lfades/next.js update-basic-auth-implementation-in-examples-ch15325 | Change |
| - | - | - | - |
| index.html gzip | 613 B | 613 B | ✓ |
| link.html gzip | 621 B | 621 B | ✓ |
| withRouter.html gzip | 608 B | 608 B | ✓ |
| Overall change | 1.84 kB | 1.84 kB | ✓ |

</details>

</details>
<hr/>
<details>
<summary><strong>Serverless Mode</strong></summary>

<br/>

<details>
<summary><strong>General</strong></summary>

|  | vercel/next.js canary  | lfades/next.js update-basic-auth-implementation-in-examples-ch15325 | Change |
| - | - | - | - |
| buildDuration | 12.6s | 12.4s | -149ms |
| nodeModulesSize | 84.9 MB | 84.9 MB | ✓ |

</details>

<details>
<summary><strong>Client Bundles (main, webpack, commons)</strong></summary>

|  | vercel/next.js canary  | lfades/next.js update-basic-auth-implementation-in-examples-ch15325 | Change |
| - | - | - | - |
| 677f882d2ed8..4972.js gzip | 12.7 kB | 12.7 kB | ✓ |
| framework.HASH.js gzip | 39 kB | 39 kB | ✓ |
| main-f1a49fb..e45e.js gzip | 6.52 kB | 6.52 kB | ✓ |
| webpack-e067..f178.js gzip | 751 B | 751 B | ✓ |
| Overall change | 58.9 kB | 58.9 kB | ✓ |

</details>

<details>
<summary><strong>Legacy Client Bundles (polyfills)</strong></summary>

|  | vercel/next.js canary  | lfades/next.js update-basic-auth-implementation-in-examples-ch15325 | Change |
| - | - | - | - |
| polyfills-4b..e242.js gzip | 31 kB | 31 kB | ✓ |
| Overall change | 31 kB | 31 kB | ✓ |

</details>

<details>
<summary><strong>Client Pages</strong></summary>

|  | vercel/next.js canary  | lfades/next.js update-basic-auth-implementation-in-examples-ch15325 | Change |
| - | - | - | - |
| _app-3b0cf13..85f8.js gzip | 1.28 kB | 1.28 kB | ✓ |
| _error-6f635..c393.js gzip | 3.44 kB | 3.44 kB | ✓ |
| hooks-d4ffc3..9e0f.js gzip | 887 B | 887 B | ✓ |
| index-17468f..5d83.js gzip | 227 B | 227 B | ✓ |
| link-b618194..5477.js gzip | 1.61 kB | 1.61 kB | ✓ |
| routerDirect..924c.js gzip | 284 B | 284 B | ✓ |
| withRouter-7..c13d.js gzip | 284 B | 284 B | ✓ |
| Overall change | 8.01 kB | 8.01 kB | ✓ |

</details>

<details>
<summary><strong>Client Build Manifests</strong></summary>

|  | vercel/next.js canary  | lfades/next.js update-basic-auth-implementation-in-examples-ch15325 | Change |
| - | - | - | - |
| _buildManifest.js gzip | 321 B | 321 B | ✓ |
| Overall change | 321 B | 321 B | ✓ |

</details>

<details>
<summary><strong>Serverless bundles</strong></summary>

|  | vercel/next.js canary  | lfades/next.js update-basic-auth-implementation-in-examples-ch15325 | Change |
| - | - | - | - |
| _error.js | 914 kB | 914 kB | ✓ |
| 404.html | 2.67 kB | 2.67 kB | ✓ |
| hooks.html | 1.92 kB | 1.92 kB | ✓ |
| index.js | 915 kB | 915 kB | ✓ |
| link.js | 973 kB | 973 kB | ✓ |
| routerDirect.js | 966 kB | 966 kB | ✓ |
| withRouter.js | 966 kB | 966 kB | ✓ |
| Overall change | 4.74 MB | 4.74 MB | ✓ |

</details>

</details>
Commit: 9ae756248129e4dd4284fe6c8dc41a0e590ffe43

File: examples/with-magic/README.md [hunk 1]
```diff
@@ -10,9 +10,9 @@ The login cookie is `httpOnly`, meaning it can only be accessed by the API, and
 
 ## Deploy your own
 
-Deploy the example using [Vercel Now](https://vercel.com/docs/now-cli#commands/overview/basic-usage):
+Once you have access to [the environment variables you'll need](#configuration), deploy the example using [Vercel](https://vercel.com?utm_source=github&utm_medium=readme&utm_campaign=next-example):
```
Comment (leerob) [inline examples/with-magic/README.md:13]: I'd say either remove the Vercel link, or make it the same link as the deploy button link 👍

File: examples/with-magic/README.md [hunk 2]
```diff
@@ -40,11 +40,30 @@ Then set each variable on `.env.local`:
...
+
+You can deploy this app to the cloud with [Vercel](https://vercel.com?utm_source=github&utm_medium=readme&utm_campaign=next-example) ([Documentation](https://nextjs.org/docs/deployment)).
+
+#### Deploy Your Local Project
+
+To deploy your local project to Vercel, push it to GitHub/GitLab/Bitbucket and [import to Vercel](https://vercel.com/import/git?utm_source=github&utm_medium=readme&utm_campaign=next-example).
+
+**Important**: When you import your project on Vercel, make sure to click on **Environment Variables** and set them to match your `.env.local` file.
+
+#### Deploy from Our Template
```
Comment (leerob) [inline examples/with-magic/README.md:65]: Is this needed - isn't it the same as above?

File: examples/with-magic/README.md [hunk 3]
```diff
@@ -40,11 +40,30 @@ Then set each variable on `.env.local`:
...
 ```bash
-vercel env add
+npm run dev
+# or
+yarn dev
 ```
+
+Your app should be up and running on [http://localhost:3000](http://localhost:3000)! If it doesn't work, post on [GitHub discussions](https://github.com/vercel/next.js/discussions).
+
+## Deploy on Vercel
```
Comment (leerob) [inline examples/with-magic/README.md:55]: Is this needed - isn't it the same as above?

File: examples/with-magic/README.md (hunk 1 above)
Comment (lfades) [reply examples/with-magic/README.md:13]: This is how we currently have it for every example 🤔

File: examples/with-magic/README.md (hunk 2 above)
Comment (lfades) [reply examples/with-magic/README.md:65]: It's similar, but more detailed. It includes the deploy button but also instructions to create a repo and link it to Vercel, useful when you use `create-next-app`.

File: examples/with-magic/README.md (hunk 3 above)
Comment (lfades) [reply examples/with-magic/README.md:55]: Same answer as for the comment above. We include a deploy button at the beginning of the example and do it again, but with more detailed instructions, at the end of the readme.

**Comment by ijjk** [comment]
# Stats from current PR

<details>
<summary><strong>Default Server Mode</strong> (Decrease detected ✓)</summary>

<br/>

<details>
<summary><strong>General</strong></summary>

|  | vercel/next.js canary  | lfades/next.js update-basic-auth-implementation-in-examples-ch15325 | Change |
| - | - | - | - |
| buildDuration | 8.8s | 8.7s | -52ms |
| nodeModulesSize | 82.6 MB | 82.6 MB | ✓ |

</details>

<details>
<summary><strong>Page Load Tests</strong> Overall decrease ⚠️</summary>

|  | vercel/next.js canary  | lfades/next.js update-basic-auth-implementation-in-examples-ch15325 | Change |
| - | - | - | - |
| / failed reqs | 0 | 0 | ✓ |
| / total time (seconds) | 1.694 | 1.738 | ⚠️ +0.04 |
| / avg req/sec | 1475.89 | 1438.1 | ⚠️ -37.79 |
| /error-in-render failed reqs | 0 | 0 | ✓ |
| /error-in-render total time (seconds) | 1.068 | 1.071 | 0 |
| /error-in-render avg req/sec | 2340.02 | 2333.58 | ⚠️ -6.44 |

</details>

<details>
<summary><strong>Client Bundles (main, webpack, commons)</strong></summary>

|  | vercel/next.js canary  | lfades/next.js update-basic-auth-implementation-in-examples-ch15325 | Change |
| - | - | - | - |
| 677f882d2ed8..5e70.js gzip | 12.8 kB | 12.8 kB | ✓ |
| framework.HASH.js gzip | 39 kB | 39 kB | ✓ |
| main-e0d2962..b163.js gzip | 6.56 kB | 6.56 kB | ✓ |
| webpack-95c2..e870.js gzip | 751 B | 751 B | ✓ |
| Overall change | 59 kB | 59 kB | ✓ |

</details>

<details>
<summary><strong>Legacy Client Bundles (polyfills)</strong></summary>

|  | vercel/next.js canary  | lfades/next.js update-basic-auth-implementation-in-examples-ch15325 | Change |
| - | - | - | - |
| polyfills-d3..23f6.js gzip | 31 kB | 31 kB | ✓ |
| Overall change | 31 kB | 31 kB | ✓ |

</details>

<details>
<summary><strong>Client Pages</strong></summary>

|  | vercel/next.js canary  | lfades/next.js update-basic-auth-implementation-in-examples-ch15325 | Change |
| - | - | - | - |
| _app-0d19cb6..5497.js gzip | 1.28 kB | 1.28 kB | ✓ |
| _error-85785..a9f3.js gzip | 3.44 kB | 3.44 kB | ✓ |
| hooks-42456f..0c06.js gzip | 887 B | 887 B | ✓ |
| index-8081ce..e44f.js gzip | 227 B | 227 B | ✓ |
| link-0ab9f83..fa00.js gzip | 1.61 kB | 1.61 kB | ✓ |
| routerDirect..c3d8.js gzip | 303 B | 303 B | ✓ |
| withRouter-0..a68e.js gzip | 302 B | 302 B | ✓ |
| Overall change | 8.05 kB | 8.05 kB | ✓ |

</details>

<details>
<summary><strong>Client Build Manifests</strong></summary>

|  | vercel/next.js canary  | lfades/next.js update-basic-auth-implementation-in-examples-ch15325 | Change |
| - | - | - | - |
| _buildManifest.js gzip | 323 B | 323 B | ✓ |
| Overall change | 323 B | 323 B | ✓ |

</details>

<details>
<summary><strong>Rendered Page Sizes</strong></summary>

|  | vercel/next.js canary  | lfades/next.js update-basic-auth-implementation-in-examples-ch15325 | Change |
| - | - | - | - |
| index.html gzip | 612 B | 612 B | ✓ |
| link.html gzip | 620 B | 620 B | ✓ |
| withRouter.html gzip | 607 B | 607 B | ✓ |
| Overall change | 1.84 kB | 1.84 kB | ✓ |

</details>

</details>
<hr/>
<details>
<summary><strong>Serverless Mode</strong></summary>

<br/>

<details>
<summary><strong>General</strong></summary>

|  | vercel/next.js canary  | lfades/next.js update-basic-auth-implementation-in-examples-ch15325 | Change |
| - | - | - | - |
| buildDuration | 10.3s | 10.4s | ⚠️ +75ms |
| nodeModulesSize | 82.6 MB | 82.6 MB | ✓ |

</details>

<details>
<summary><strong>Client Bundles (main, webpack, commons)</strong></summary>

|  | vercel/next.js canary  | lfades/next.js update-basic-auth-implementation-in-examples-ch15325 | Change |
| - | - | - | - |
| 677f882d2ed8..5e70.js gzip | 12.8 kB | 12.8 kB | ✓ |
| framework.HASH.js gzip | 39 kB | 39 kB | ✓ |
| main-e0d2962..b163.js gzip | 6.56 kB | 6.56 kB | ✓ |
| webpack-95c2..e870.js gzip | 751 B | 751 B | ✓ |
| Overall change | 59 kB | 59 kB | ✓ |

</details>

<details>
<summary><strong>Legacy Client Bundles (polyfills)</strong></summary>

|  | vercel/next.js canary  | lfades/next.js update-basic-auth-implementation-in-examples-ch15325 | Change |
| - | - | - | - |
| polyfills-d3..23f6.js gzip | 31 kB | 31 kB | ✓ |
| Overall change | 31 kB | 31 kB | ✓ |

</details>

<details>
<summary><strong>Client Pages</strong></summary>

|  | vercel/next.js canary  | lfades/next.js update-basic-auth-implementation-in-examples-ch15325 | Change |
| - | - | - | - |
| _app-0d19cb6..5497.js gzip | 1.28 kB | 1.28 kB | ✓ |
| _error-85785..a9f3.js gzip | 3.44 kB | 3.44 kB | ✓ |
| hooks-42456f..0c06.js gzip | 887 B | 887 B | ✓ |
| index-8081ce..e44f.js gzip | 227 B | 227 B | ✓ |
| link-0ab9f83..fa00.js gzip | 1.61 kB | 1.61 kB | ✓ |
| routerDirect..c3d8.js gzip | 303 B | 303 B | ✓ |
| withRouter-0..a68e.js gzip | 302 B | 302 B | ✓ |
| Overall change | 8.05 kB | 8.05 kB | ✓ |

</details>

<details>
<summary><strong>Client Build Manifests</strong></summary>

|  | vercel/next.js canary  | lfades/next.js update-basic-auth-implementation-in-examples-ch15325 | Change |
| - | - | - | - |
| _buildManifest.js gzip | 323 B | 323 B | ✓ |
| Overall change | 323 B | 323 B | ✓ |

</details>

<details>
<summary><strong>Serverless bundles</strong></summary>

|  | vercel/next.js canary  | lfades/next.js update-basic-auth-implementation-in-examples-ch15325 | Change |
| - | - | - | - |
| _error.js | 1 MB | 1 MB | ✓ |
| 404.html | 2.67 kB | 2.67 kB | ✓ |
| hooks.html | 1.92 kB | 1.92 kB | ✓ |
| index.js | 1 MB | 1 MB | ✓ |
| link.js | 1.06 MB | 1.06 MB | ✓ |
| routerDirect.js | 1.05 MB | 1.05 MB | ✓ |
| withRouter.js | 1.05 MB | 1.05 MB | ✓ |
| Overall change | 5.16 MB | 5.16 MB | ✓ |

</details>

</details>
Commit: a8fe1d540dd2d2a611c65dee17050d701f4e359c

**Review by leerob — APPROVED** [review]
Thank you for updating these!

**Comment by ijjk** [comment]
# Stats from current PR

<details>
<summary><strong>Default Server Mode</strong> (Increase detected ⚠️)</summary>

<br/>

<details>
<summary><strong>General</strong></summary>

|  | vercel/next.js canary  | lfades/next.js update-basic-auth-implementation-in-examples-ch15325 | Change |
| - | - | - | - |
| buildDuration | 9.3s | 9.5s | ⚠️ +133ms |
| nodeModulesSize | 82.6 MB | 82.6 MB | ✓ |

</details>

<details>
<summary><strong>Page Load Tests</strong> Overall increase ✓</summary>

|  | vercel/next.js canary  | lfades/next.js update-basic-auth-implementation-in-examples-ch15325 | Change |
| - | - | - | - |
| / failed reqs | 0 | 0 | ✓ |
| / total time (seconds) | 1.973 | 1.82 | -0.15 |
| / avg req/sec | 1266.92 | 1373.75 | +106.83 |
| /error-in-render failed reqs | 0 | 0 | ✓ |
| /error-in-render total time (seconds) | 1.191 | 1.132 | -0.06 |
| /error-in-render avg req/sec | 2098.48 | 2208.71 | +110.23 |

</details>

<details>
<summary><strong>Client Bundles (main, webpack, commons)</strong></summary>

|  | vercel/next.js canary  | lfades/next.js update-basic-auth-implementation-in-examples-ch15325 | Change |
| - | - | - | - |
| 677f882d2ed8..5e70.js gzip | 12.8 kB | 12.8 kB | ✓ |
| framework.HASH.js gzip | 39 kB | 39 kB | ✓ |
| main-e0d2962..b163.js gzip | 6.56 kB | 6.56 kB | ✓ |
| webpack-95c2..e870.js gzip | 751 B | 751 B | ✓ |
| Overall change | 59 kB | 59 kB | ✓ |

</details>

<details>
<summary><strong>Legacy Client Bundles (polyfills)</strong></summary>

|  | vercel/next.js canary  | lfades/next.js update-basic-auth-implementation-in-examples-ch15325 | Change |
| - | - | - | - |
| polyfills-d3..23f6.js gzip | 31 kB | 31 kB | ✓ |
| Overall change | 31 kB | 31 kB | ✓ |

</details>

<details>
<summary><strong>Client Pages</strong></summary>

|  | vercel/next.js canary  | lfades/next.js update-basic-auth-implementation-in-examples-ch15325 | Change |
| - | - | - | - |
| _app-0d19cb6..5497.js gzip | 1.28 kB | 1.28 kB | ✓ |
| _error-85785..a9f3.js gzip | 3.44 kB | 3.44 kB | ✓ |
| hooks-42456f..0c06.js gzip | 887 B | 887 B | ✓ |
| index-8081ce..e44f.js gzip | 227 B | 227 B | ✓ |
| link-0ab9f83..fa00.js gzip | 1.61 kB | 1.61 kB | ✓ |
| routerDirect..c3d8.js gzip | 303 B | 303 B | ✓ |
| withRouter-0..a68e.js gzip | 302 B | 302 B | ✓ |
| Overall change | 8.05 kB | 8.05 kB | ✓ |

</details>

<details>
<summary><strong>Client Build Manifests</strong></summary>

|  | vercel/next.js canary  | lfades/next.js update-basic-auth-implementation-in-examples-ch15325 | Change |
| - | - | - | - |
| _buildManifest.js gzip | 323 B | 323 B | ✓ |
| Overall change | 323 B | 323 B | ✓ |

</details>

<details>
<summary><strong>Rendered Page Sizes</strong></summary>

|  | vercel/next.js canary  | lfades/next.js update-basic-auth-implementation-in-examples-ch15325 | Change |
| - | - | - | - |
| index.html gzip | 612 B | 612 B | ✓ |
| link.html gzip | 620 B | 620 B | ✓ |
| withRouter.html gzip | 607 B | 607 B | ✓ |
| Overall change | 1.84 kB | 1.84 kB | ✓ |

</details>

</details>
<hr/>
<details>
<summary><strong>Serverless Mode</strong></summary>

<br/>

<details>
<summary><strong>General</strong></summary>

|  | vercel/next.js canary  | lfades/next.js update-basic-auth-implementation-in-examples-ch15325 | Change |
| - | - | - | - |
| buildDuration | 11s | 11.4s | ⚠️ +460ms |
| nodeModulesSize | 82.6 MB | 82.6 MB | ✓ |

</details>

<details>
<summary><strong>Client Bundles (main, webpack, commons)</strong></summary>

|  | vercel/next.js canary  | lfades/next.js update-basic-auth-implementation-in-examples-ch15325 | Change |
| - | - | - | - |
| 677f882d2ed8..5e70.js gzip | 12.8 kB | 12.8 kB | ✓ |
| framework.HASH.js gzip | 39 kB | 39 kB | ✓ |
| main-e0d2962..b163.js gzip | 6.56 kB | 6.56 kB | ✓ |
| webpack-95c2..e870.js gzip | 751 B | 751 B | ✓ |
| Overall change | 59 kB | 59 kB | ✓ |

</details>

<details>
<summary><strong>Legacy Client Bundles (polyfills)</strong></summary>

|  | vercel/next.js canary  | lfades/next.js update-basic-auth-implementation-in-examples-ch15325 | Change |
| - | - | - | - |
| polyfills-d3..23f6.js gzip | 31 kB | 31 kB | ✓ |
| Overall change | 31 kB | 31 kB | ✓ |

</details>

<details>
<summary><strong>Client Pages</strong></summary>

|  | vercel/next.js canary  | lfades/next.js update-basic-auth-implementation-in-examples-ch15325 | Change |
| - | - | - | - |
| _app-0d19cb6..5497.js gzip | 1.28 kB | 1.28 kB | ✓ |
| _error-85785..a9f3.js gzip | 3.44 kB | 3.44 kB | ✓ |
| hooks-42456f..0c06.js gzip | 887 B | 887 B | ✓ |
| index-8081ce..e44f.js gzip | 227 B | 227 B | ✓ |
| link-0ab9f83..fa00.js gzip | 1.61 kB | 1.61 kB | ✓ |
| routerDirect..c3d8.js gzip | 303 B | 303 B | ✓ |
| withRouter-0..a68e.js gzip | 302 B | 302 B | ✓ |
| Overall change | 8.05 kB | 8.05 kB | ✓ |

</details>

<details>
<summary><strong>Client Build Manifests</strong></summary>

|  | vercel/next.js canary  | lfades/next.js update-basic-auth-implementation-in-examples-ch15325 | Change |
| - | - | - | - |
| _buildManifest.js gzip | 323 B | 323 B | ✓ |
| Overall change | 323 B | 323 B | ✓ |

</details>

<details>
<summary><strong>Serverless bundles</strong></summary>

|  | vercel/next.js canary  | lfades/next.js update-basic-auth-implementation-in-examples-ch15325 | Change |
| - | - | - | - |
| _error.js | 1 MB | 1 MB | ✓ |
| 404.html | 2.67 kB | 2.67 kB | ✓ |
| hooks.html | 1.92 kB | 1.92 kB | ✓ |
| index.js | 1 MB | 1 MB | ✓ |
| link.js | 1.06 MB | 1.06 MB | ✓ |
| routerDirect.js | 1.05 MB | 1.05 MB | ✓ |
| withRouter.js | 1.05 MB | 1.05 MB | ✓ |
| Overall change | 5.16 MB | 5.16 MB | ✓ |

</details>

</details>
Commit: 11ef9182eb70fafd8b6ebf60d146776cc145a21c

**Pull request merged**